bdd_tests/features/cleanup_old_records.feature

```diff
@@ -0,0 +1,129 @@
+Feature: Ability to clean up old records stored in database
+
+
+  Scenario: Check the ability to clean up old records from `new_reports` table if the table is empty.
```

Comment:
this scenario already exists in cleanup_new_records.feature file

Reply:
yes, but ideally scenarios should be independent as it is possible to run just selected one (selected by tags etc. etc.)
which reminds me - add tags!